### ISSUE TITLE
SMTPSenderRefused on welcome emails

### ISSUE THREAD

**Issue by colegleason** [acm-uiuc/liquid#295]
It seems CITES SMTP does not like our from address.

**Comment by colegleason** [comment]
If the address is `ACM@UIUC <acm@acm.illinois.edu>` then the server thinks `ACM@UIUC` is an address and complains that the sender does not exist.

I have tried to send emails with just `acm@acm.illinois.edu` but they only went through once. I wonder if it is being caught as spam somewhere and not being passed on. It isn't showing up in my spam folders.

I also tried:

* `acm@illinois.edu`
* `acm@uiuc.edu`
* `ACM <acm@acm.illinois.edu>`
* `ACM at UIUC <acm@acm.illinois.edu>`

No luck.

@CJS7070 @ajmadsen any ideas?

**Comment by colegleason** [comment]
Emailed CITES about this.

**Comment by colegleason** [comment]
Who knows what the issue is here. @mvd7793 looked into this for KSP and found that CITES rejects emails for pretty much no reason at all.

Using a SMTP host setup on the host by @nhandler. Would love to migrate to a central ACM mail server if one is created in the future (@ajmadsen).

**Comment by mvd7793** [comment]
Disclaimer: This could be entirely unrelated.

I looked into a similar error exactly for Splash actually! A sample of the bottom of a stack trace is included below. In the Splash code, I could find absolutely no way that we should have tried to send an email to `mather` by itself. It should enforce validation with an @ sign, and did in every way I tried. Maybe it's related to the mailserver somehow. Hasn't happened again.

    File "/usr/lib/python2.7/smtplib.py", line 733, in sendmail
    raise SMTPRecipientsRefused(senderrs)
    SMTPRecipientsRefused: {u'mather': (501, '<mather>: recipient address must contain a domain')}
